### PR TITLE
PS-9079 Release notes for Percona Server for MySQL 8.3.0-1

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   .github/workflows/main.yml
	new file:   docs/release-notes/8.3.0-1.md
	modified:   docs/release-notes/release-notes-index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.3.0-1.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+* The EXPLAIN command provides improved output for queries involving JSON data to better understand query execution plans.
+
+## Deprecation or removal
+
+A future release may remove deprecated features, variables and options. The usage of these deprecated items may cause a warning. We recommend migrating from deprecated variables and options as soon as possible.
+
+This release removes the following features:
+
+* MyISAM tables are no longer supported.
```
Comment (patrickbirch) [inline docs/release-notes/8.3.0-1.md]: I think it's still supported.

Comment (alina-derkach-oaza) [reply docs/release-notes/8.3.0-1.md]: rewritten

**Review by patrickbirch — APPROVED** [review]
Good

File: docs/release-notes/8.3.0-1.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+[PS-9018](https://perconadev.atlassian.net/browse/PS-9018): The replica stalls when `log_slave_updates=0` and a Data Definition Language (DDL) operation is executed against a non-replicated database on that replica.
+
+[PS-9048](https://perconadev.atlassian.net/browse/PS-9048): In the Debug build, assets with OPTIMIZE table and fulltext indexes failed.
+
+[PS-9117](https://perconadev.atlassian.net/browse/PS-9117): The server exited after setting the `innodb_interpreter_output` system variable.
+
+[PS-9075](https://perconadev.atlassian.net/browse/PS-9075): `ALTER TABLE ... ALGORITHM=INPLACE` failed for table which is no longer encrypted.
+
+[PS-9044](https://perconadev.atlassian.net/browse/PS-9044): Added the following variables to MyRocks:
+
```
Comment (percona-ysorokin) [inline docs/release-notes/8.3.0-1.md:55]: Please add
https://perconadev.atlassian.net/browse/PS-9125
as well

File: docs/release-notes/8.3.0-1.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.3.0-1] includes all the features and bug fixes available in the
+MySQL 8.3 Community Edition in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+This release merges the MySQL 8.3 code base. Improvements and bug fixes introduced by Oracle for MySQL 8.3 and included in Percona Server for MySQL are the following:
+
+* Implements tagged GTIDs (Global Transaction Identifiers) to group related transactions. This helps enhance replication management and simplify tracking transaction dependencies.
```
Comment (percona-ysorokin) [inline docs/release-notes/8.3.0-1.md]: Also our Binary Log UDFs were adapted to work properly with Tagged GTIDs
https://github.com/percona/percona-server/commit/92cb30d2fcca18ad86f387259af4a2b4e520ea3d
`Make sure Binlog Utils UDFs are compatible with new GTID format (tagged GTIDs)` item from the merge TODO list

File: docs/release-notes/8.3.0-1.md (hunk 2 above)
Comment (percona-ysorokin) [reply docs/release-notes/8.3.0-1.md:55]: And
https://perconadev.atlassian.net/browse/PS-9083

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.3.0-1.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.3.0-1.md]: Added

File: docs/release-notes/8.3.0-1.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.3.0-1.md:55]: Added

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**